client/object_search.go

```diff
@@ -242,17 +273,22 @@ func (c *Client) ObjectSearchInit(ctx context.Context, containerID cid.ID, prm P
 
 	err = signServiceMessage(signer, &req)
 	if err != nil {
-		return nil, fmt.Errorf("sign request: %w", err)
+		err = fmt.Errorf("sign request: %w", err)
+		return nil, err
 	}
 
 	var r ObjectListReader
 	ctx, r.cancelCtxStream = context.WithCancel(ctx)
 
-	r.stream, err = rpcapi.SearchObjects(&c.c, &req, client.WithContext(ctx))
+	r.stream, err = rpcAPISearchObjects(&c.c, &req, client.WithContext(ctx))
 	if err != nil {
-		return nil, fmt.Errorf("open stream: %w", err)
+		err = fmt.Errorf("open stream: %w", err)
+		return nil, err
 	}
 	r.client = c
+	r.statisticCallback = func(duration time.Duration, err error) {
+		c.sendStatistic(stat.MethodObjectSearch, duration, err)
```

Comment:
Is it counted like a `MethodObjectSearch` now in `pool`? I'd expect some separate name since it's not a real method, rather just reading from the stream.

Reply:
Same with get/put.

Reply:
Extended with three new `Method*`